plugins_test.go

```diff
@@ -70,6 +70,46 @@ func TestPluginInstallUninstallOfficialRegistry(t *testing.T) {
 	}
 }
 
+func TestPluginInstallWithProgressBar(t *testing.T) {
```

Comment:
Worth showing this to @gailazar300 , just to make sure it doesn't conflict with anything she's currently working on on the jfrog-cli plugins track.